src/widgets/gauge.rs

```diff
@@ -129,6 +154,21 @@ impl<'a> Widget for Gauge<'a> {
     }
 }
 
+fn get_unicode_block<'a>(frac: f64) -> &'a str {
+    match (frac * 8.0).round() as u16 {
+        //get how many eighths the fraction is closest to
+        1 => block::ONE_EIGHTH,
+        2 => block::ONE_QUARTER,
+        3 => block::THREE_EIGHTHS,
+        4 => block::HALF,
+        5 => block::FIVE_EIGHTHS,
+        6 => block::THREE_QUARTERS,
+        7 => block::SEVEN_EIGHTHS,
+        8 => "",
```

Comment:
shouldn't this be a full block ?

Reply:
There isn't a difference, I changed it over so it makes more sense with readability